test/test_2d.jl

```diff
@@ -395,6 +395,43 @@ end
       end
     end
   end
+
+  @testset "Subcell limiting coefficients" begin
+    isdir(outdir) && rm(outdir, recursive=true)
+    run_trixi(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_sedov_blast_wave_sc_subcell.jl"), maxiters=10)
+
+    @timed_testset "without reinterpolation" begin
+      # Create and test output without reinterpolation
+      @test_nowarn trixi2vtk(joinpath(outdir, "solution_000010.h5"), output_directory=outdir, reinterpolate=false)
+      outfilename = "solution_000010.vtu"
+      out_file = joinpath(outdir, outfilename)
+
+      # save output file to `artifacts` to facilitate debugging of failing tests
+      testname = "2d-treemesh-shockcapturing-subcell-no-reinterp"
+      cp(out_file, joinpath(artifacts_dir, testname * "-" * outfilename), force=true)
+
+      # remote file path is actually a URL so it always has the same path structure
+      remote_filename = "2d/treemesh/dgsem_sedov_subcell_no_interp_10.vtu"
+      ref_file = get_test_reference_file("dgsem_sedov_subcell_no_interp_10.vtu", remote_filename)
+      compare_point_data(out_file, ref_file)
+    end
+
+    @timed_testset "with reinterpolation" begin
+      # Create and test output without reinterpolation
+      @test_nowarn trixi2vtk(joinpath(outdir, "solution_000010.h5"), output_directory=outdir, reinterpolate=true)
+      outfilename = "solution_000010.vtu"
+      out_file = joinpath(outdir, outfilename)
+
+      # save output file to `artifacts` to facilitate debugging of failing tests
+      testname = "2d-treemesh-shockcapturing-subcell-reinterp"
+      cp(out_file, joinpath(artifacts_dir, testname * "-" * outfilename), force=true)
+
+      # remote file path is actually a URL so it always has the same path structure
+      remote_filename = "2d/treemesh/dgsem_sedov_subcell_interp_10.vtu"
+      ref_file = get_test_reference_file("dgsem_sedov_subcell_interp_10.vtu", remote_filename)
+      compare_cell_data(out_file, ref_file)
```

Comment:
Is `compare_cell_data` the correct function call here? Just wondering, I actually don't know.

Reply:
I also was wondering about that. I wrote all my thoughts to this "discussion" https://github.com/trixi-framework/Trixi2Vtk.jl/pull/67#discussion_r1576160118. Did you read that?

Reply:
Right. Interpolation is to piecewise constant VTK cells, while original data is high-order Lagrangian VTK cells.